README.md

```diff
@@ -206,6 +312,51 @@ This table tells us:
   which is faster, because the difference was too small. 1000 iterations could
   be as much as 13% faster, or as much as 21% slower, than 1001 iterations.
 
+## Confidence intervals
+
+Loosely speaking, a confidence interval is a range of plausible values for a
+parameter like runtime, and the _confidence level_ (which tachometer always
+fixes to _95%_) corresponds to the degree of confidence we have that interval
+contains the _true value_ of that parameter.
+
+More precisely speaking, the 95% confidence level describes the _long-run
+proportion of confidence intervals that will contain the true value_.
```

Comment:
I'm not sure I follow that definition, but may that's ok 'cause it's the precise definition?